R/roxygenize.R

```diff
@@ -30,16 +30,10 @@ roxygenize <- function(package.dir = ".",
                        load_code = NULL,
                        clean = FALSE) {
 
-  base_path <- normalizePath(package.dir, mustWork = TRUE)
```

Comment:
I wonder if we should still call `normalizePath()`?

Reply:
What are the advantages? Keeping a relative path makes the paths in feedback much more useful.

Reply:
At least on Windows `normalizePath()` does various conversions, so if we don't call it now, then the behaviour probably changes in some cases.

OTOH if it is only used for reporting then that is fine.

Reply:
Hmmm, if I re-add it, none of the tests fail, so I must've only needed it in an interim state. Seems better to keep to minimise unintended consequences.